codex/conf.nim

```diff
@@ -125,6 +126,13 @@ type
       abbr: "d"
       name: "data-dir" }: OutDir
 
+    metaDir* {.
+      desc: "The directory where codex will store metadata"
+      defaultValue: DefaultMetaDataDir
+      defaultValueDesc: $DefaultMetaDataDir
+      abbr: "md"
+      name: "meta-dir" }: OutDir
```

Comment:
Out of curiosity, why does the metadata need its own directory? Could we not use the exisiting data dir?

Reply:
It doesn't need to. I defaulted it to a sub-dir of the dataDir. But maybe one could imagine a situation where you'd like to store metadata on a drive with different properties (speed? seek times?) than data.

Reply:
I would also check potential security issues, i.e. possibility of reading/writing metadata by the request for raw data. Even if a correct app will never allow it, it may increase an attack vector.